src/Device/DeviceSelector/DeviceList/MoreDeviceInfo.tsx

```diff
@@ -60,9 +60,11 @@ const MoreDeviceInfo: FC<{ device: Device }> = ({ device }) => (
         <Row>
             <MaybeDeviceName device={device} />
         </Row>
-        <Row>
-            <Serialports ports={device.serialPorts} />
-        </Row>
+        {device.serialPorts ? (
```

Comment:
Alternatively `{device.serialPorts && <></>}` as opposed to `{device.serialPorts ? <></> : null}`